test/data/testvib.py

```diff
@@ -207,6 +239,13 @@ def testfreeenergy(self):
          """Is the freeenergy reasonable"""
          self.assertAlmostEqual(-381.90808120060200, self.data.freeenergy, self.freeenergy_places)
 
+
+class Psi4IRTest(GenericIRTest):
+    """Customized vibrational frequency unittest"""
+
+    max_force_constant = 9.37

```

Comment:
Can you explain why this is necessary? Is the geometry different from other packages?

Reply:
As I add cooment, it is because that Psi4 IR test data is calculated with RHF instead of B3LYP.